graylog2-server/src/test/java/org/graylog/plugins/views/search/elasticsearch/searchtypes/ESMessageListTest.java

```diff
@@ -88,6 +92,47 @@ public void appliesDecoratorsToQueryStringIfHighlightingActivated() {
         JsonPathAssert.assertThat(doc).jsonPathAsString("$.highlight.highlight_query.query_string.query").isEqualTo("Foobar!");
     }
 
+    @Test
+    public void passesTypeOfSortingFieldAsUnmappedType() {
+        final MessageList messageList = someMessageListWithSorting();
+        final ESGeneratedQueryContext context = mockQueryContext(messageList);
+        when(context.fieldTypes(Collections.singleton("stream1"))).thenReturn(Collections.singletonMap("somefield", Collections.singleton("long")));
```

Comment:
It's unclear at first glance where "stream1" and "someField" come from.
I'd suggest either declaring them in variables at the start of test and passing them down or extracting them from the messageList object.

Reply:
I decided to pass them to the method generating the message list, so they are close to the assertion.